packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml

```diff
@@ -1901,6 +1910,9 @@ spec:
                 return $SaToken
             }
 
+            # Disable firewall temporarily for SSH and other internal ports access
+            Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
```

Comment:
We don't need to reenable the firewall at any point, or does `antrea.ps1` handle that?

Reply:
we don't reenable it back, there's a bigger refactoring of this part including the correct fw rules.